packages/libs/react-ui/src/components/Form/TextField/TextField.tsx

```diff
@@ -12,18 +13,26 @@ export const TextField: FC<ITextFieldProps> = ({
   disabled = false,
   inputProps,
   status,
-  ...rest
+  ...props
 }) => {
   const { id } = inputProps;
+  const ref = useRef(null);
+  const { labelProps, inputProps: ariaInputProps } = useTextField(props, ref);
```

Comment:
there are description and error props as well, so we need to use them to have a completely accessible formfiled